src/Native/System.Native/pal_io.h

```diff
@@ -113,6 +113,46 @@ enum class NodeType : int16_t
 };
 
 /**
+ * Constants from sys/file.h for lock types
+ */
+enum class LockOperations : int32_t
```

Comment:
We should have static asserts that verify these constants all match the platform values, right?